install.sh

```diff
@@ -421,7 +421,7 @@ cp -r swagger-ui/dist "$rprefix"/swagger-ui
 install -d -m755 -d "$rprefix"/api
 cp -r api/api-doc "$rprefix"/api
 install -d -m755 -d "$rprefix"/scyllatop
-cp -r tools/scyllatop/* "$rprefix"/scyllatop
+cp -r scyllatop/* "$rprefix"/scyllatop
```

Comment:
I don't understand this change - how come what used to be tools/scyllatop/ is now in scyllatop/? Where did we change it?

Reply:
I modified scyllatop location on relocatable package, but I wil revert it to follow @fruch 's advice, so nevermind